### PR TITLE
views: redirect old pages

### PR DESCRIPTION
**Pull request by atrisovic**
* Addresses: #1914

Signed-off-by: Ana Trisovic <ana.trisovic@gmail.com>

### CODE REVIEW

**Comment by atrisovic** [comment]
**note:** I have also here deleted the old `about` pages

File: cernopendata/modules/pages/views.py [hunk 1]
```diff
@@ -178,40 +178,21 @@ def about_breadcrumbs():
 
 
 @blueprint.route('/about')
-@register_breadcrumb(blueprint, '.about', _('About'))
 def about():
     """Render view for general about page."""
-    return render_template('cernopendata_pages/about/index.html')
+    return render_template('/docs/cern-open-data-portal')
```
Comment (ioannistsanaktsidis) [inline cernopendata/modules/pages/views.py]: I think redirect instead of render_template

File: cernopendata/modules/pages/views.py [hunk 2]
```diff
@@ -178,40 +178,21 @@ def about_breadcrumbs():
...
-
-@blueprint.route('/about/cms')
-@register_breadcrumb(blueprint, '.about.about_cms', 'CMS')
-def about_cms():
-    """Render about cms template."""
-    return redirect('/docs/about-cms')
+@blueprint.route('/about/<exp>')
+def about_exp(exp):
+    """Render about <experiment> pages."""
+    return render_template('/docs/about-%s' % exp)
```
Comment (ioannistsanaktsidis) [inline cernopendata/modules/pages/views.py]: Also here redirect. And better ```return redirect('/docs/about-{}'.format(exp))```

Comment (atrisovic) [reply cernopendata/modules/pages/views.py]: ok fixed 👍

**Comment by ioannistsanaktsidis** [comment]
cc @tiborsimko

**Comment by tiborsimko** [comment]
@atrisovic The redirections do not work fully on the latest `master` for me, e.g. `/about/ALICE` is one example.  

See also the remaining instances of the `about_alice` etc views used in:

```console
$ git grep 'cernopendata_pages.*about_'
cernopendata/templates/cernopendata_theme/header.html:          <li><a class="dropdown-item" href="{{url_for('cernopendata_pages.about_alice')}}">ALICE Open Data</a></li>
cernopendata/templates/cernopendata_theme/header.html:          <li><a class="dropdown-item" href="{{url_for('cernopendata_pages.about_atlas')}}">ATLAS Open Data</a></li>
cernopendata/templates/cernopendata_theme/header.html:          <li><a class="dropdown-item" href="{{url_for('cernopendata_pages.about_cms')}}">CMS Open Data</a></li>
cernopendata/templates/cernopendata_theme/header.html:          <li><a class="dropdown-item" href="{{url_for('cernopendata_pages.about_lhcb')}}">LHCb Open Data</a></li>
cernopendata/templates/cernopendata_theme/header.html:          <li><a class="dropdown-item" href="{{url_for('cernopendata_pages.about_opera')}}">OPERA Open Data</a></li>
cernopendata/templates/cernopendata_theme/header_index.html:          <li><a class="dropdown-item" href="{{url_for('cernopendata_pages.about_alice')}}">ALICE Open Data</a></li>
cernopendata/templates/cernopendata_theme/header_index.html:          <li><a class="dropdown-item" href="{{url_for('cernopendata_pages.about_atlas')}}">ATLAS Open Data</a></li>
cernopendata/templates/cernopendata_theme/header_index.html:          <li><a class="dropdown-item" href="{{url_for('cernopendata_pages.about_cms')}}">CMS Open Data</a></li>
cernopendata/templates/cernopendata_theme/header_index.html:          <li><a class="dropdown-item" href="{{url_for('cernopendata_pages.about_lhcb')}}">LHCb Open Data</a></li>
cernopendata/templates/cernopendata_theme/header_index.html:          <li><a class="dropdown-item" href="{{url_for('cernopendata_pages.about_opera')}}">OPERA Open Data</a></li>
```

that we need to update to `about_exp` and friends...

Can you please rebase against latest `master` and recheck everything?

**Pull request merged**